src/ontology/mondo-edit.obo

```diff
@@ -20509,6 +20509,7 @@ property_value: IAO:0000589 "myopia (disease)" xsd:string
 id: MONDO:0001385
 name: cortical blindness
 def: "Visual impairment due to visual cortex dysfunction." [NCIT:P378]
+synonym: "cortical visual impairment" EXACT []
```

Comment:
need dbxref

Reply:
fixed in https://github.com/monarch-initiative/mondo/pull/7335